src/components/UniversalNavbar/UniversalNavbar.js

```diff
@@ -194,15 +194,18 @@ class UniversalNavbar extends React.Component {
 
 UniversalNavbar.propTypes = {
   /** Hide cta on mobile viewport */
-  hideMobileCta: PropTypes.bool.isRequired,
+  hideMobileCta: PropTypes.bool,
+  /** Hide cta on desktop */
+  hideDesktopCta: PropTypes.bool,
   /** Run analytics function when CTA Button gets clicked */
-  trackCtaClick: PropTypes.func.isRequired,
+  trackCtaClick: PropTypes.func,
```

Comment:
I think this should be required, as long as defaultProps is defined.
From the site [React doc site](https://reactjs.org/docs/typechecking-with-proptypes.html):
> The propTypes typechecking happens after defaultProps are resolved, so typechecking will also apply to the defaultProps.

Reply:
what's the point of marking them required if they have defaults?

Reply:
i'll just do it but i am actually curious what you think about this

Reply:
You have a point. Never thought about that. There was this ESLint rule I used before, where everything had to be required or be in defaultProps. You have a point.

Reply:
> what's the point of marking them required if they have defaults?


Inversely, there's no reason to add a default if  something is required. For example:
[this eslint](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/require-default-props.md) states:

> Enforce a defaultProps definition for every prop **that is not a required prop** (react/require-default-props)

So I'd like us to remove the default values for required it's just adding more noise.